build.gradle

```diff
@@ -129,12 +137,22 @@ subprojects {
         testImplementation "org.mockito:mockito-core:$mockitoVersion"
         testImplementation "org.mockito:mockito-junit-jupiter:$mockitoVersion"
 
+        testImplementation "org.awaitility:awaitility:$awaitilityVersion"
+
         testRuntimeOnly "org.slf4j:slf4j-log4j12:$slf4jVersion"
     }
 
     tasks.named('test') {
         // Use junit platform for unit tests.
         useJUnitPlatform()
+        testLogging {
+            events 'passed', 'skipped', 'failed'
+            showStandardStreams = true
+            showExceptions = true
+            showStackTraces = true
+            showCauses = true
+            exceptionFormat "full"
+        }
```

Comment:
AFAIU the idea is to see the test logs during in CI for easier debugging? Enabling these by default and for all tests including the unit ones looks a bit cumbersome. Maybe we can enable these only for e2e or even be ok with just `./gradlew e2e:test -i` in the CI specifically?

Reply:
I can move these to e2e. `-i` haven't got the level of details I needed when troubleshooting

Reply:
Done